### ISSUE TITLE
Inline code blocks are not indexed

### ISSUE THREAD

**Issue by axilleas** [algolia/docsearch#828]
## Description

Some inline code blocks are not searchable

## Steps to reproduce

1. Go to https://docs.gitlab.com
1. Enter `SAST_DISABLED` into the search box and run the search.

## Actual result

The mention of `SAST_DISABLED` from [this page](https://docs.gitlab.com/ee/topics/autodevops/#customizing-gitlab-ciyml) **does not** appear in results.

## Expected result

The mention of `SAST_DISABLED` from [this page](https://docs.gitlab.com/ee/topics/autodevops/#customizing-gitlab-ciyml) appears in results

## Further details

1. The same buggy behavior is not true for all similar mentions from that page. For instance, `AUTO_DEVOPS_CHART_REPOSITORY` is being found, while `CANARY_REPLICAS` and `CANARY_PRODUCTION_REPLICAS` are not. Maybe the reason is the Algolia search index mapping setup.
1. When portions of the documentation text from the [page](https://docs.gitlab.com/ee/topics/autodevops/#customizing-gitlab-ciyml) are searched for, they appear in search results.

## Config details

- We're using the latest docsearch version from jsdelivr
- Our configuration file https://github.com/algolia/docsearch-configs/blob/master/configs/gitlab.json
- Downstream issue https://gitlab.com/gitlab-org/gitlab-docs/issues/343

**Comment by Haroenv** [comment]
Might be related to the config here: https://github.com/algolia/docsearch-configs/blob/master/configs/gitlab.json

**Comment by s-pace** [comment]
👋 @axilleas 

Thanks for the details. 

@Haroenv was correct, we needed to update your selectors to make sure `SAST_DISABLED` was matching one of those. Since  `SAST_DISABLED` is only wrapped into a `td` tag, it was not pickup.

> The same buggy behavior is not true for all similar mentions from that page. For instance, AUTO_DEVOPS_CHART_REPOSITORY is being found, while CANARY_REPLICAS and CANARY_PRODUCTION_REPLICAS are not. Maybe the reason is the Algolia search index mapping setup.

These elements were pick up from your table of content. However, it is not recommended to indexed these elements since it can be seen as noise. Table of content are only a reference to the result and not the result itself. See 

The update of your configuration solved this by:
- Scrapping every elements of your table
- Make sure `_` is indexed
- Make sure we do not index table of content.

I have indexed your website and everything seems to works as expected.
![image](https://user-images.githubusercontent.com/32097720/64674687-a9ff8900-d471-11e9-8843-32c7b2a56820.png)
